src/gwt/src/org/rstudio/core/rebind/command/ShortcutsEmitter.java

```diff
@@ -67,8 +67,8 @@ public void generate(SourceWriter writer) throws UnableToCompleteException
 
          String condition = childEl.getAttribute("if");
          String command = childEl.getAttribute("refid");
-         String shortcutValue = childEl.getAttribute("value");
-         String title = childEl.getAttribute("title");
+         String shortcutValue = childEl.getAttribute("value"); //TODO: i18n
+         String title = "###" + childEl.getAttribute("title"); //TODO: i18n
```

Comment:
Looks like this is work-in-progress related to shortcut localization. Probably wasn't intended to be in this PR?

Better if this PR is ONLY property files.